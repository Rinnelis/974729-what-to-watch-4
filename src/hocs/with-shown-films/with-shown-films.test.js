import React from "react";
import renderer from "react-test-renderer";
import withShownFilms from "./with-shown-films.jsx";

const MockComponent = () => <div />;
const MockComponentWrapped = withShownFilms(MockComponent);

it(`withShownFilms is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      maxShownFilms={8}
      onShownFilmsAmountReset={() => {}}
      onShownFilmsAdd={() => {}}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
