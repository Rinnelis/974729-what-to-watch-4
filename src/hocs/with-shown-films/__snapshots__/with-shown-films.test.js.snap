// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withShownFilms is rendered correctly 1`] = `<div />`;
